packages/scheduler-sequelize/src/helpers.ts

```diff
@@ -144,17 +147,12 @@ const updateFinishTask = async (job?: JobInstance | null) => {
  */
 export const timestampHelperFactory =
   (jobScheduler: JobScheduler): TimestampHelper =>
-  <T = any, R = any>(job: JobModel, task: TaskCallback<T, R>) =>
+  <T extends { context?: JobContext } = any, R = any>(
+    job: JobModel,
+    task: TaskCallback<T, R>
+  ) =>
   async (args: T, context: JobContext): Promise<any> => {
-    if (!context) {
-      try {
-        return task(args, context);
-      } catch (e) {
-        return null;
-      }
-    }
-
-    const jobId = context.job?.id;
+    const jobId = args.context?.job?.id ?? context?.job?.id;
```

Comment:
does the full job get serialized into the meta of the task being run?

Reply:
yes, I attached the image for ref
![image (1)](https://github.com/ordermentum/steveo/assets/19221138/f83268d5-543d-40c2-8160-5143e5cdab98)

Reply:
We should make the job object leaner. 
https://github.com/ordermentum/steveo/blob/C1-777-fix-jobid/packages/scheduler-prisma/src/index.ts#L334-L336

```typescript
await task(item.data, {
       job: {
          id: item.id
       }
});
```
The above change will make it leaner and should not break things.
We'll need to do a sweep of what service uses the context object (if any). 

Not saying we do it here, we should def unblock this. A new ticket also works.

Reply:
done creating a ticket